### PR TITLE
rewrite server in ts, use ts-node

### PR DESCRIPTION
**Pull request by GoldinGuy**
- Please review this before merging, no idea if it will work in production. Works locally

### CODE REVIEW

File: server.ts [hunk 1]
```diff
@@ -0,0 +1,135 @@
+require("dotenv").config();
+var sslRedirect = require("heroku-ssl-redirect");
+
+// Twillio config
+var twillioAuthToken =
```
Comment (JosephSemrai) [inline server.ts]: These are at the root level. Should we change the vars to consts?

File: server.ts [hunk 2]
```diff
@@ -0,0 +1,135 @@
...
+				room,
+				" already full with ",
+				numClients,
+				"clients in the room."
+			);
+			socket.emit("full", room);
+		}
+	});
+
+	socket.on("test", () => {
```
Comment (JosephSemrai) [inline server.ts]: Probably should remove the test handler lol

File: server.ts [hunk 3]
```diff
@@ -0,0 +1,135 @@
...
+	socket.on("answer", (answer: string, room: string, uuid: string) => {
+		console.log(
+			"Answer from " + socket.id + "... emitting to " + uuid + "in room: ",
+			room
+		);
+		io.to(uuid).emit("answer", answer, socket.id);
+	});
+});
+
+var port = process.env.PORT || 3001;
```
Comment (JosephSemrai) [inline server.ts]: Const

File: server.ts [hunk 4]
```diff
@@ -0,0 +1,135 @@
...
+var path = require("path");
+const app = express();
+
+// Live reload for development
+if (process.env.NODE_ENV === "development") {
+	const livereload = require("livereload");
+	const connectLivereload = require("connect-livereload");
+
+	// open livereload high port and start to watch public directory for changes
+	const liveReloadServer = livereload.createServer();
```
Comment (JosephSemrai) [inline server.ts]: Livereload isn't necessary anymore for the React version

**Review by JosephSemrai — APPROVED** [review]

File: server.ts (hunk 1 above)
Comment (GoldinGuy) [reply server.ts]: probably

**Comment by GoldinGuy** [comment]
Made requested changes, also cleaned up comments and fixed typos. Again, no idea if this will work in production, but seems fine in dev, so we should be good.

**Review by JosephSemrai — APPROVED** [review]

**Comment by JosephSemrai** [comment]
LGTM!

**Pull request merged**